src/util/graph.h

```diff
@@ -584,6 +598,58 @@ std::vector<typename N::node_indext> grapht<N>::get_reachable(
   return result;
 }
 
+/// Run recursive depth-limited search on the graph, starting
+/// from a vector of start nodes, to find the nodes reachable within n steps,
+/// excluding the start node unless a self loop exists.
+/// \param from : vector of node indices to start from
+/// \param depth : depth limit
+/// \return set of node indices of reachable nodes
```

Comment:
I have no idea why Doxygen can't cope with this one, or at least I can't see anything wrong with it. You might try moving the documentation to the declarations above to see whether that makes Doxygen happier.

Reply:
Would you mind trying to move the documentation to the declaration? It seems Doxygen is still refusing to play along.

Reply:
that's weird, because get_reachable is documented in the same way and in the same place. The only difference is get_reachable doesn't document the return